cmake/Format.cmake

```diff
@@ -53,7 +53,7 @@ if(LANG STREQUAL c)
   endif()
 elseif(LANG STREQUAL lua)
   list(FILTER changed_files INCLUDE REGEX "\\.lua$")
-  list(FILTER changed_files INCLUDE REGEX "^runtime/")
+  list(FILTER changed_files INCLUDE REGEX "^(runtime|scripts)/")
```

Comment:
Even with this change, `make formatlua` doesn't work for me. I had to manually use:

    stylua --color=always --respect-ignores scripts/